dbfit-java/config/checkstyle/checkstyle.xml

```diff
@@ -9,7 +9,9 @@
     <module name="UnusedImports"/>
   </module>
   <module name="FileTabCharacter"/>
-  <module name="NewlineAtEndOfFile"/>
+  <module name="NewlineAtEndOfFile">
+   <property name="lineSeparator" value="lf" />
```

Comment:
Indentation on this line is a but unusual (check the rest sub-tags in the xml file).